app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -194,12 +198,12 @@ def get_floating_ips
     end
 
     def floating_ips_neutron
-      @network_service.floating_ips
+      @network_service.handled_list(:floating_ips)
     end
 
     # maintained for legacy nova network support
     def floating_ips_nova
-      @network_service.addresses_for_accessible_tenants
+      @connection.handled_list(:addresses)
```

Comment:
Well, the original implementation was calling `@network_service`, but if it was in this method, then `@network_service` was really nova.  So, the call didn't really fail, it was just calling something that wasn't neutron.  It's better in my mind to keep is specific like you've got here.